### PR TITLE
Add SmoothOperator

### PR DESCRIPTION
**Pull request by turbaszek**
Easter is coming so I just came with idea of an harmless easter egg 😄 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Will it actually log any outpit :) ? I think not as Dummy Opertor's execute is not well.. executed :)?

**Review by potiuk — APPROVED** [review]

**Comment by uranusjr** [comment]
This operator may be too criminal to use in production

**Comment by uranusjr** [comment]
Yes the `inherits_from_dummy_operator` check would kick in to not run `execute`.

**Review by potiuk — CHANGES_REQUESTED** [review]
> Yes the `inherits_from_dummy_operator` check would kick in to not run `execute`.

Well. Then I think we can't merge it because it DOES NOT WORK AS ADVERTISED!

**Review by potiuk — APPROVED** [review]
Now - you talking to me :)

**Comment by potiuk** [comment]
But even smooth operator needs some :heart:  when it comes to static checks :D

File: airflow/operators/smooth.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class SmoothOperator(BaseOperator):
+    """
+    Operator that does literally nothing but it logs YouTube link to
+    Sade song "Smooth Operator".
+
+    The task is evaluated by the scheduler but never processed by the executor.
+    """
+
+    ui_color = '#e8f7e4'
+    inherits_from_dummy_operator = False
```
Comment (eladkal) [inline airflow/operators/smooth.py]: Now that it's no longer inherit from `DummyOperator`
do we still need `inherits_from_dummy_operator`?

Comment (uranusjr) [reply airflow/operators/smooth.py]: No, this is automatically False

Comment (turbaszek) [reply airflow/operators/smooth.py]: Good catch, fixed! 👍

**Comment by potiuk** [comment]
Static checks are Sade, ehm SAD.

**Review by uranusjr — APPROVED** [review]

**Comment by potiuk** [comment]
Not SADE any more and just random failure. Merging :)

**Comment by potiuk** [comment]
Just in time for 2.3.0.

We could not wait for SmoothOperator. It's been highly requested feature by many.

**Review by potiuk — APPROVED** [review]

**Pull request merged**